static/src/js/components/GranuleResults/__tests__/GranuleResultsHeader.test.js

```diff
@@ -295,4 +307,33 @@ describe('granuleFilters link', () => {
       expect(props.onToggleSecondaryOverlayPanel).toHaveBeenCalledWith(false)
     })
   })
+
+  describe('when the g key is pressed', () => {
+    test('toggles the granule panel state correctly', () => {
```

Comment:
Can you add one more test that shows `onToggleSecondaryOverlayPanel` does not get called with a different `location.pathname`, `/search` for example

Reply:
Added test for pathname.